kube-up

```diff
@@ -251,15 +251,24 @@ function ntp_updater(){
   local TIME_GAP="$2"
   local RESULT
 
-  until ntpdate "${K8SUP_NTPSERVER_CLUSTER_IP}" 2>/dev/null; do
-    sleep 1
-  done
-  echo "NTP client synchronized!" 1>&2
-  sleep "${TIME_GAP}"
+  if ! ps aux | grep -v grep | grep -q "/etc/chrony/chrony.conf"; then
```

Comment:
Is check /etc/chrony/chrony.conf string in ps -aux a good way to determine any NTP server run on this node?
How about check port 123(NTP)?

Reply:
It's a cluster service that deployed by k8sup and it doesn't use hostNetwork and NodePort.

Reply:
I got it, just try to determine whether ntp server pod runs on the same node or not by checking host IP of pod next time.